### PR TITLE
Add Boxy SVG case study

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add the Boxy SVG case study.
- The review happened in the associated [Google doc](https://docs.google.com/document/d/10NmH51BaWXxc_QCB5HvjS4j60BZYuxZppZ-VMFtRiuE/edit?resourcekey=0-_a9J35awUaGbnUDsESju6A).

### CODE REVIEW

**Comment by tomayac** [comment]
FYI: @jarek-foksa

**Review by rachelandrew — APPROVED** [review]

**Comment by jarek-foksa** [comment]
While reading the article on the staging site, I realized there is a bug in the code snippet that I provided (and thus in the app itself). To be more specific, this fragment:

```js
    } else if (subfamily.includes("semibold")) {
      face = "600";
    } else if (subfamily.includes("bold")) {
      face = "700";
    } else if (subfamily.includes("extrabold")) {
      face = "800";
    } else if (subfamily.includes("ultrabold")) {
      face = "900";
    }
```

... should be replaced with:

```js
    } else if (subfamily.includes("semibold")) {
      face = "600";
    } else if (subfamily.includes("extrabold")) {
      face = "800";
    } else if (subfamily.includes("ultrabold")) {
      face = "900";
    } else if (subfamily.includes("bold")) {
      face = "700";
    }
```

File: site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      face = "500";
+    } else if (subfamily.includes("semibold")) {
+      face = "600";
+    } else if (subfamily.includes("bold")) {
+      face = "700";
+    } else if (subfamily.includes("extrabold")) {
+      face = "800";
+    } else if (subfamily.includes("ultrabold")) {
+      face = "900";
+    }
```
Comment (tomayac) [inline site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md:106]:
```suggestion
    } else if (subfamily.includes("semibold")) {
      face = "600";
    } else if (subfamily.includes("extrabold")) {
      face = "800";
    } else if (subfamily.includes("ultrabold")) {
      face = "900";
    } else if (subfamily.includes("bold")) {
      face = "700";
    }
```

File: site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+date: 2023-01-31
+# updated: 2023-01-31
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/81OfVmaxz3gTsqVIXu5i.png
+alt: Boxy SVG icon pattern.
+authors:
+  - thomassteiner
+  - jarekfoksa
+tags:
+  - capabilities
+  - case-study
```
Comment (tomayac) [inline site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md:15]:
```suggestion
  - case-study
  - fugu-case-study
```

File: site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  - fugu-case-study
+---
+
+[Boxy SVG](https://boxy-svg.com/) is a vector graphics editor. Its main use case is editing drawings in the SVG file format, for creating illustrations, logos, icons, and other elements of graphic design. It's developed by Polish developer [Jarosław Foksa](https://foksa.name/) and was initially released on March 15, 2013. Jarosław runs a [Boxy SVG blog](https://boxy-svg.com/blog/) in which he announces new features he adds to the app. The developer is a strong supporter of [Chromium's Project Fugu](/capabilities/) and even has a [Fugu tag](https://boxy-svg.com/ideas?tag=Fugu) on the app's ideas tracker.
```
Comment (tomayac) [inline site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md:19]:
```suggestion
## Introduction

(This article is also available in form of a video.)

{% YouTube id="Y40vMQap9fs", startTime="611" %}

[Boxy SVG](https://boxy-svg.com/) is a vector graphics editor. Its main use case is editing drawings in the SVG file format, for creating illustrations, logos, icons, and other elements of graphic design. It's developed by Polish developer [Jarosław Foksa](https://foksa.name/) and was initially released on March 15, 2013. Jarosław runs a [Boxy SVG blog](https://boxy-svg.com/blog/) in which he announces new features he adds to the app. The developer is a strong supporter of [Chromium's Project Fugu](/capabilities/) and even has a [Fugu tag](https://boxy-svg.com/ideas?tag=Fugu) on the app's ideas tracker.
```

File: site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'How vector image editing app Boxy SVG uses the Local Font Access API to let users pick their favorite local fonts'
+subhead: >
+  The Local Font Access API provides a mechanism to access the user's locally installed font data, including higher-level details such as names, styles, and families, as well as the raw bytes of the underlying font files. Learn how the SVG editing app Boxy SVG makes use of this API.
+date: 2023-01-31
+# updated: 2023-01-31
```
Comment (tomayac) [inline site/en/blog/how-boxysvg-uses-the-local-font-access-api/index.md:7]:
```suggestion
date: 2023-06-01
# updated: 2023-06-01
```

**Pull request merged**